tests/test_devshell.py

```diff
@@ -107,7 +116,7 @@ class _AuthReferenceServer(threading.Thread):
     def __init__(self, response=None):
         super(_AuthReferenceServer, self).__init__(None)
         self.response = (response or
-                         '["joe@example.com", "fooproj", "sometoken"]')
+            '["joe@example.com", "fooproj", "sometoken", ' + str(EXPIRES_IN) + ']')
```

Comment:
Thanks for doing the re-factor with the global, but sadly now this line is longer than 80 chars. (Sorry to be the PEP8 enforcer)